tests/integration/ceph_upgrade_test.go

```diff
@@ -135,7 +136,26 @@ func (s *UpgradeSuite) testUpgrade(useHelm bool, initialCephVersion v1.CephVersi
 
 	s.verifyOperatorImage(installer.LocalBuildTag)
 	s.verifyRookUpgrade(numOSDs)
-	err := s.installer.WaitForToolbox(s.namespace)
+
+	// From L140 to L156 we are just restarting the rook-ceph-toolbox pod so that it has the latest ceph username form secret
+	// rook-ceph-mon. Above mentioned line should be removed later once we aremove the support of rook 1.9
+	toolboxLabel := "app=rook-ceph-tools"
```

Comment:
As part of the new backward compatibility approach, let's not update the toolbox during the upgrade